packages/block-editor/src/components/line-height-control/index.js

```diff
@@ -37,7 +37,7 @@ export default function LineHeightControl( {
 		 * Step up/down actions can be triggered by keydowns of the up/down arrow keys,
 		 * or by clicking the spin buttons.
 		 */
-		switch ( nextValue ) {
+		switch ( `${ nextValue }` ) {
```

Comment:
What bug is this change fixing? Was the bug caused by the removal of `composeStateReducers` ?

Reply:
The cause is an change in order of the composed reducers. With this PR, the reducer for `LineHeightControl` runs after the reducer for `NumberControl`. So here,`nextValue` comes from the `state.value` of the preceding reducer and coming from `NumberControl` that **may be** a number which was throwing off the switch statement here checking for strings. This https://github.com/WordPress/gutenberg/issues/38845#issuecomment-1060943133 might help clarify.